src/clamav_report/clamav_report.py

```diff
@@ -213,23 +187,17 @@ def create_host_row(host_results):
     """Create a row of data from a host's results."""
     facts = host_results["Gathering Facts"][0]["ansible_facts"]
     # extract the mtimes from the "stat" module invocations
-    mtimes = dict()
+    mtimes = {}
     for stat_task in host_results["stat"]:
         path = stat_task["invocation"]["module_args"]["path"]
         mtime = stat_task["stat"].get("mtime", 0)  # 0 if it doesn't exist
         mtimes[path] = timestamp_to_string(mtime)
     row = {key: None for key in FIELDS}
-    row["IP_ADDR1"] = facts["ansible_default_ipv4"]["address"]
-    row["COMPUTER_NAME"] = facts["ansible_hostname"]
-    row["OPERATION_SYSTEM"] = facts["ansible_system"]
-    row["MAJOR_VERSION"] = 0
-    row["MINOR_VERSION"] = 0
-    row["AGENT_VERSION"] = "0.0.0.0"  # nosec
-    row["CIDS_ENGINE_VERSION"] = "0.0.0.0"  # nosec
-    row["CIDS_DRV_ONOFF"] = "ON"
-    row["LAST_SCAN_TIME"] = mtimes[LAST_SCAN_LOG_FILENAME]
-    row["LAST_VIRUS_TIME"] = mtimes[LAST_DETECTION_FILENAME]
-    row["PATTERNDATE"] = mtimes[CLAMAV_DB_FILENAME]
+    row["System Name"] = facts["ansible_hostname"]
+    row["Last Update Time"] = mtimes[CLAMAV_DB_FILENAME]
+    row["Last Scan Time"] = mtimes[LAST_SCAN_LOG_FILENAME]
+    # row["Last Detection Time"] = mtimes[LAST_DETECTION_FILENAME]
+    row["Host IPS Status (Host IPS)"] = "ON"
```

Comment:
Where does the value for `Group Name` get set?  I'm not seeing it in this file.

Reply:
You are correct- it is being left blank.  I will add a comment to that effect.

Reply:
OK, I figured it was something like that.  Thank you!